build.xml

```diff
@@ -209,7 +209,12 @@
     <property name="reports" value="${build}/reports"/>
 
     <mkdir dir="${build}/ant-elfanalyser" />
-    <javac classpath="lib/ant.jar" destdir="${build}/ant-elfanalyser" includeantruntime="false">
+    <javac classpath="lib/ant.jar" 
+           destdir="${build}/ant-elfanalyser" 
+           includeantruntime="false"
+           source="1.6"
```

Comment:
Just curious: Should this use something like: ${compatibility} instead of hardcoded 1.6 ?